### PR TITLE
Update deps + sync next.jdbc's RowBuilder

### PR DESCRIPTION
**Pull request by janosmeszaros**
A fix for #11 

Noticed some perf degradation after bumping the version of `next.jdbc`, could be some internal changes there:

1. [next.jdbc: porsas compiled](https://github.com/metosin/porsas/blob/b52322a9647b30419f79672c8f776daa6ad81e9e/test/porsas/core_test.clj#L91): from ~1200ns to ~1800ns
2. [next.jdbc](https://github.com/metosin/porsas/blob/b52322a9647b30419f79672c8f776daa6ad81e9e/test/porsas/core_test.clj#L96): from ~2400ns to ~3200ns

### CODE REVIEW

File: src/porsas/next.clj [hunk 1]
```diff
@@ -23,9 +23,17 @@
...
            rs/ResultSetBuilder
            (->rs [_] (transient []))
            (with-row [_ rs row] (conj! rs row))
-           (rs! [_ rs] (persistent! rs))))))))
+           (rs! [_ rs] (persistent! rs))
+           clojure.lang.ILookup ; only supports :cols and :rsmeta
+           (valAt [this k] (get this k nil))
+           (valAt [this k not-found]
+             (case k
+               :cols   (map last (pj/col-map rs key))
```
Comment (janosmeszaros) [inline src/porsas/next.clj:37]: Could this be an empty vector as `column-count` is constantly `0`?

**Review by imrekoszo — APPROVED** [review]

File: src/porsas/next.clj (hunk 1 above)
Comment (seancorfield) [reply src/porsas/next.clj:37]: The only thing `column-count` is used for internally is iterating over the (number of columns) in the internal `row-builder` function, which I don't believe is called in the case of the `porsas` code? (or maybe it is done to short-circuit that method so much of the internals of `next.jdbc` can be skipped over?)

The actual column names do exist and `(map last (pj/col-map rs key))` looks like the way `porsas` fetches those when building the row.

Comment (janosmeszaros) [reply src/porsas/next.clj:37]: I believe this `row-builder` exists to do some optimizations on how `next.jdbc` builds up the results (at least that's my understanding based on the test [here](https://github.com/metosin/porsas/blob/0a803db1e689a120babe7b11138e4b866c715050/test/porsas/core_test.clj#L93)). So if the `column-count` is constantly `0` then the `cols` here won't be used so no need to calculate them, although it is lazy so won't be calculated unnecessarily.

**Comment by ikitommi** [comment]
Hopefully this was done, thanks!!

**Pull request merged**